src/SectionTitle.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cx from 'classnames';
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+export const SECTIONTITLE_CLASS = 'sectionTitle';
+/**
+ * @module SectionTitle
+ */
+class SectionTitle extends React.Component {
+	render() {
+		const {
+			action,
+			className,
+			title,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			SECTIONTITLE_CLASS,
+			className
+		);
+
+		return (
+			<Flex
+				align='center'
+				className={classNames}
+				{...other}
+				>
+				<FlexItem>
```

Comment:
actually, no - that would override the display property of `Chunk` (the chunks would become flex children). Wrapping the whole thing could work, but I don't want to overload a `Chunk` with the section title class, which may also mess with the `display` style property